doc/sphinx/source/community/release_strategy.rst

```diff
@@ -248,6 +248,7 @@ These are the detailed steps to take to make a release.
 
    - A release branch is created and branch protection rules are set up so only the release manager (i.e. the person in charge of the release branch) can push commits to that branch.
    - Make a release candidate with the release branch following the :ref:`ESMValCore release instructions <esmvalcore:how-to-make-a-release>`.
+   - Uncomment the release candidate channel item (i.e. ``conda-forge/label/esmvalcore_rc``) in the ``environment.yml`` of ESMValTool to add it to the list of channels used. Adjust the pin on ESMValCore after each release candidate (e.g. ``esmvalcore==2.8.0rc1``).
```

Comment:
I'm no expert on this, but couldn't you just update the pin so it uses `esmvalcore` 2.8 directly? Then there would be no need to update it so often

Reply:
I am with Bouwe on that one. We need to update the pin to 2.8 for the final release anyway, and if we have multiple RCs than we only need to update this once.

Reply:
@schlunma pls read me comment below - we need to make sure the env gets built with the correct RC for testing - conda envs are as fussy as Lancia cars - we may risk testing with RC1 being unaware while we think we are testing with RC2

Reply:
I would hope that mamba/conda is smart enough to always use the latest RC (after everything has settled on conda-forge after the release), but sure, that way it's safer. Just  thought we could avoid more work for the release manager that way :sweat_smile:

Reply:
indeed - chances are favourable for that to happen - but here's an example when this could go south real fast: one of our dependencies gets updated after a while, and creates a conflict, mamba will diligently install an older version of it that may trigger downgrades to a number of other packages, including our baby Core. Let's be German here, for once :grin: :de:

Reply:
I agree with V. Even if we don't pin esmvalcore so striclty, the release manager needs to check that the environment creation works fine and contains the right version of esmvalcore. For testing purposes, it might be simpler to have a strict pin. I added a note that the RM should check the env creation.